core/src/main/java/com/workday/autoparse/json/parser/JsonParserUtils.java

```diff
@@ -929,10 +931,13 @@ public static JSONArray parseAsJsonArray(JsonReader reader, String key) throws I
                 @SuppressWarnings("unchecked")
                 T castItem = (T) o;
                 parsedItem = castItem;
-            } else if (o instanceof JSONObject) {
+            }  else if (o instanceof JSONObject) {
                 parsedItem = convertJsonObject((JSONObject) o, typeClass, itemParser, context);
             } else if (o instanceof String && converter != null) {
                 parsedItem = converter.convert((String) o);
+            } else if (o == null) { //The parsed array has an explicit null, so add a null to the collection
```

Comment:
Put this on its own line.
```
} else if (o == null) {
   // comment
   collection.add(null);
   continue;
}
```
Also, grammar nazi says to put a period at the end of the commend and space after the `//`. :-)

Reply:
Done and done haha